src/ex.rs

```diff
@@ -307,6 +199,39 @@ pub fn frob_tomls(ex: &Experiment, crates: &[Crate]) -> Result<()> {
     Ok(())
 }
 
+pub fn capture_shas<DB: ExperimentResultDB>(crates: &[Crate], db: &DB) -> Result<()> {
+    for krate in crates {
+        if let Crate::GitHub(ref repo) = *krate {
+            let url = repo.url();
+            let dir = gh_mirrors::repo_dir(&url)?;
+            let r = RunCommand::new("git", &["log", "-n1", "--pretty=%H"])
```

Comment:
Can we use `git rev-parse HEAD` here? I think that should be much cleaner and potentially faster...

Reply:
Done.